src/routes/dictionaries/+page.svelte

```diff
@@ -87,7 +59,7 @@
       sequenceAdaptationName = '';
     } catch (e) {
       createDictionaryError = (e as Error).message;
-      showFailureToast('Command Dictionary Create Failed');
+      showFailureToast('Command Dictionary or Sequence Adaptation Create Failed');
```

Comment:
I think you can move this final toast in with the error handling of the effects function as well.

Reply:
got it, I moved it over to effect.ts as well